docs/samples/sklearn/README.md

```diff
@@ -16,7 +16,7 @@ dump(clf, 'model.joblib')
 Then, we can run the Scikit-learn Server using the generated model and test for prediction. Models can be on local filesystem, S3 compatible object storage or Google Cloud Storage.
 
 ```shell
-python -m sklearnserver --model_dir model.joblib --model_name svm
+python -m sklearnserver --model_dir file://model.joblib --model_name svm
```

Comment:
Honestly, I'd prefer if the "recommended" usage for the python server just used relative paths. This file convention is less and less common these days. I'm happy to support it, but maybe not recommend it.

Reply:
Maybe we can stick to "only" relative path, but relative path can start with "/" OR "."